docs/build-scripts.md

```diff
@@ -0,0 +1,13 @@
+# Build Scripts
+
+As part of the deployment process, HM Cloud supports running the running of tasks to handle pre-deployment tasks such as minifying JavaScript, running webpack, compiling SASS, etc. The specific build process is defined in a file `.build-script` in the root of your project's git repository.
```

Comment:
Might benefit some people to show an example here, as well as elaborate a bit on the available shebangs for different languages eg. `#!/usr/bin/env php`